turbopuffer/namespace.py

```diff
@@ -3,9 +3,9 @@
 import json
 from datetime import datetime
 from turbopuffer.error import APIError
-from turbopuffer.vectors import Cursor, VectorResult, VectorColumns, VectorRow, batch_iter, b64encode_vector
+from turbopuffer.vectors import Cursor, VectorResult, VectorColumns, VectorRow, b64decode_vector, batch_iter, b64encode_vector
```

Comment:
not a big deal at all! but i have to be honest and speak my mind: something about `b64decode_vector` and `b64encode_vector` not being next to each other bothers me 😂

Reply:
Haha, me too, now that you mention it! Turns out `b64decode_vector` didn't end up needing to be imported in the first place. Removed.

Reply:
oh lol great!